src/jit/core_templates.expr

```diff
@@ -689,6 +689,14 @@
       (carg $2 ptr)
       (carg \$0 ptr))))
 
+(template: objprimspec
+  (if
+    (nz $1)
+    (cast
+      (^getf (^storage_spec $1) MVMStorageSpec boxed_primitive)
+      int_sz (&sizeof MVMuint16) (&QUOTE MVM_JIT_CAST_UNSIGNED))
```

Comment:
`(&QUOTE MVM_JIT_CAST_UNSIGNED)` can be written as `cast_unsigned` (the `MVM_JIT_` is an implied prefix); the `(&QUOTE)` macro expression is intended for when this isn't suitable.